### PR TITLE
Examples in the docs

### PR DESCRIPTION
**Pull request by kagarmoe**
### Description

We need to follow docs best practices and standardize on a set example information for domains, ips, names, emails, addresses, etc.
There's currently a lot of real information, which could land us in trouble.

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/style_guide.md [hunk 1]
```diff
@@ -565,14 +587,89 @@ like "Create a new user" (by its very nature, a user created **is** a
...
+Here is a list of some example names for you to use (all of the last names are translations of "Chef", according to the internet):
+
+Ares Koch
+Tamira Bucatar
+Dan Gotvach
+Lena Chushi
+Haris Shefu
+Booker Yolisa
+Kala Baavarchee
+Samuel Tagaluto
```
Comment (IanMadd) [inline content/style_guide.md]:
```suggestion
- Ares Koch
- Tamira Bucatar
- Dan Gotvach
- Lena Chushi
- Haris Shefu
- Booker Yolisa
- Kala Baavarchee
- Samuel Tagaluto
```

File: content/style_guide.md [hunk 2]
```diff
@@ -565,14 +587,89 @@ like "Create a new user" (by its very nature, a user created **is** a
...
+
+### Example Names
+
+Don't reveal personal information in examples, such as the names of real people, real email addresses, or phone numbers.
+
+Don't use the names of bands, musicians, or characters from works that are under copyright.
+
+When writing about security, follow the accepted convention and use "Alice" and "Bob". Following this convention helps readers see that they are reading a topic about security and integrate the Chef information with their existing knowledge.
+
+Here is a list of some example names for you to use (all of the last names are translations of "Chef", according to the internet):
```
Comment (IanMadd) [inline content/style_guide.md]: I'm not sure I understand what "all of the last names are translations of "Chef", according to the internet" means, but ok 🤷

File: content/style_guide.md [hunk 3]
```diff
@@ -565,14 +587,89 @@ like "Create a new user" (by its very nature, a user created **is** a
...
+### Example Email Addresses
+
+akoch@example.com
+tvucatar@example.com
+dgotvach@example.com
+lchushi@example.com
+hshefu@example.com
+byolisa@example.com
+kbaavarchee@example.com
+stagaluto@example.com
```
Comment (IanMadd) [inline content/style_guide.md]:
```suggestion
- akoch@example.com
- tvucatar@example.com
- dgotvach@example.com
- lchushi@example.com
- hshefu@example.com
- byolisa@example.com
- kbaavarchee@example.com
- stagaluto@example.com
```

File: content/style_guide.md (hunk 2 above)
Comment (kagarmoe) [reply content/style_guide.md]: Literally, the magic of Google translate.

File: content/style_guide.md [hunk 4]
```diff
@@ -462,7 +484,7 @@ For example:
       {{</* /foundation_tabs_panel */>}}
     {{</* /foundation_tabs_panels */>}}
 
-#### Parameters
+#### Panel Parameters
```
Comment (IanMadd) [inline content/style_guide.md:487]: We should probably change "Parameters" on line 439 to "Tab Parameters" to match this.

File: content/style_guide.md [hunk 5]
```diff
@@ -565,14 +587,89 @@ like "Create a new user" (by its very nature, a user created **is** a
...
+
+Use fictional street addresses in examples.
+
+Use these fictional addresses:
+
+2943 Conifer Drive
+Seattle, WA 98122
+
+1214 Hollow Road
+Boston, MA 02110
```
Comment (IanMadd) [inline content/style_guide.md:642]: It turns out, adding two spaces after a line followed by another line keeps the two lines in the same paragraph, but adds a `<br>` between the lines. Who knew.

```suggestion
2943 Conifer Drive  
Seattle, WA 98122

1214 Hollow Road  
Boston, MA 02110
```

Comment (kagarmoe) [reply content/style_guide.md:642]: The wonders of Hugo.

**Comment by IanMadd** [comment]
There is also juliachild@chef.io in the README.md and community_contributions.md.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**